src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1139,14 +1139,45 @@ public static bool IsParameterTypeContext(this SyntaxTree syntaxTree, int positi
         {
             tokenOnLeftOfPosition = tokenOnLeftOfPosition.GetPreviousTokenIfTouchingWord(position);
 
-            if (tokenOnLeftOfPosition.IsKind(SyntaxKind.OpenParenToken))
+            if (tokenOnLeftOfPosition.Parent.IsKind(SyntaxKind.ParenthesizedExpression,
```

Comment:
what if the token on left of the position is the close paren token?    will this prevent something like ```(Foo()) $$is```?